zkevm-circuits/src/evm_circuit/execution/begin_tx.rs

```diff
@@ -187,6 +187,7 @@ impl<F: Field> ExecutionGadget<F> for BeginTxGadget<F> {
             code_source: To(code_hash.expr()),
             gas_left: To(gas_left),
             reversible_write_counter: To(2.expr()),
+            log_id: To(0.expr()),
```

Comment:
From here it seems `log_id` is zero indexed.

Reply:
this is by design, no logs in the begin tx, hence set it to zero